nixos/modules/services/misc/workout-tracker.nix

```diff
@@ -0,0 +1,121 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+
+with lib;
+
+let
+  inherit (lib) types;
+  cfg = config.services.workout-tracker;
+  exe = lib.getExe cfg.package;
```

Comment:
As this attribute is just used once, just write `${lib.getExe cfg.package}` there.